api/apps/brand_partners/index.coffee

```diff
@@ -0,0 +1,11 @@
+express = require 'express'
+routes = require './routes'
+{ setUser, authenticated, adminOnly } = require '../users/routes'
+
+app = module.exports = express()
+
+app.get '/brand-partners', routes.index
```

Comment:
Hmm no strong opinion here, keeping it dashes on the client works for me.

In general dashes are blessed as the more "SEO friendly" urls b/c Google used to not treat underscores as word separators, but that 1. Obviously doesn't matter for a non-website app like Writer and 2. I think Google treats both as separators  these days anyways.